src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -128,16 +175,15 @@ public static Timespan parse(final String s) {
 				
 			}
 		}
+
 		return new Timespan(t);
 	}
 	
-	private final long millis;
-	
 	public Timespan() {
 		millis = 0;
 	}
 	
-	public Timespan(final long millis) {
+	public Timespan(long millis) {
```

Comment:
Can't comment directly at it, but I think it might be good to have a constructor (or could be a method) like
```java
Timespan(TimePeriod period, long time)
```
This could replace the `fromTicks` static method and one would just do something like `new Timespan(TimePeriod.TICKS, 50)`